# Gene Sequence Alignment: Leveraging AI for Accurate Comparison

## Abstract: 
  This project focuses on analyzing genomic sequence alignment, specifically targeting conserved
regions and functional domains within the COX1 gene of Homo sapiens, Chimpanzee & House
Mice. Sequence alignment is essential for identifying homologous sequences across species,
shedding light on evolutionary relationships and biological function. By employing both the
**Needleman-Wunsch** and **Smith-Waterman** algorithms and **Clustering**, this study explores the
nuances of global and local sequence alignment, respectively. The Needleman-Wunsch algorithm 
seeks optimal alignment over entire sequences, capturing broad similarities, while the SmithWaterman 
algorithm highlights highly similar subregions, revealing conserved domains that may
indicate functional significance. We utilize real genomic datasets in **FASTA** format obtained from
**GenBank of NCBI**, ensuring that our study relies on authentic data. This approach enhances our
analysis by providing a realistic view of genomic similarity, as observed in natural gene
sequences. Through this dual-method alignment, we aim to demonstrate how these conserved
regions serve as markers of evolutionary proximity and genetic relevance between closely related
species. Additionally, insights gained from the similarities between these COX1 genes could
contribute to broader studies on gene functionality and evolutionary lineage in related species.
This projectâ€™s results underscore the effectiveness of sequence alignment techniques for
evolutionary biology research and open possibilities for further exploration in functional
genomics.
